Decred Wallet/Features/Security/RequestPinViewController.swift

```diff
@@ -17,93 +16,130 @@ class RequestPinViewController: SecurityBaseViewController {
     var onUserEnteredPin: ((_ pin: String) -> Void)?
     
     var requestPinConfirmation = false
-    var pinToConfirm: String = ""
     
-    @IBOutlet weak var cancelBtn: UIButton!
+    var submitEnabled: Bool = false {
+        didSet {
+            if self.submitEnabled == true {
+                self.btnCommit.isEnabled = true
+                btnCommit.setBackgroundColor(UIColor.appColors.decredBlue, for: .normal)
+            }else {
+                self.btnCommit.isEnabled = false
+                btnCommit.setBackgroundColor(UIColor.appColors.darkGray, for: .normal)
```

Comment:
It should be possible to specify different background colors for different button states: enabled and disabled. If so, this `submitEnabled` variable would be unnecessary and it'll suffice to just set `self.btnCommit.isEnabled = true/false`.